### ISSUE TITLE
Unused Resource in the application

### ISSUE THREAD

**Issue by chamika** [systers/malaria-app-android#42]
These resources can be removed or replaced if used in the application in a hard coded way
Following strings from strings.xml are not used and can be removed
app_homescreen_name
second_analytic_screen_title
cash_label
medication_label
remind_label
location_spinner_label
medication_spinner_label
items_spinner_label
user_medicine_settings_activity_drug_array
location_array
item_array

Following colors from colors.xml can be removed
view_pager_indicator_text_color
user_medicine_second_text_color
symptom_color_selected
reaction_color_selected
static_text_color
journal_entry_listitem_text_color
note_text_disabled
blue
navy
sky
lightgray02
dark
red
lighter_green

Following resource files are not using
/malaria-app-android/res/drawable/accept_medi_normal_tiny.jpg
/malaria-app-android/res/drawable/add_b.png
/malaria-app-android/res/drawable/appicon.png
/malaria-app-android/res/drawable/appicon_small.png
/malaria-app-android/res/drawable/cal_left_arrow_on.png
/malaria-app-android/res/drawable/cal_right_arrow_on.png
/malaria-app-android/res/drawable/calendar.9.png
/malaria-app-android/res/drawable/calendar_centralheader.png
/malaria-app-android/res/drawable/calendar_days.png
/malaria-app-android/res/drawable/calendar_header.svg
/malaria-app-android/res/drawable/calendar_tile.png
/malaria-app-android/res/drawable/calendar_top_header.png
/malaria-app-android/res/drawable/calendar_view.png
/malaria-app-android/res/drawable/settings_large.png
/malaria-app-android/res/drawable/topbar.png
/malaria-app-android/res/drawable/trip_spinner.png
/malaria-app-android/res/drawable/trip_spinner_arr.png
/malaria-app-android/res/drawable/trip_spinner_bg.xml
/malaria-app-android/res/drawable/trip_spinner_dte.png
/malaria-app-android/res/drawable/delete_b.png
/malaria-app-android/res/drawable-hdpi/ic_launcher.png
/malaria-app-android/res/drawable/icon.png
/malaria-app-android/res/drawable/peacecorps_small.png
/malaria-app-android/res/drawable/placeholder_analytic_text.png
/malaria-app-android/res/drawable/reject_medi_normal_tiny.jpg
/malaria-app-android/res/drawable-hdpi/bg.png

/malaria-app-android/res/layout/screeen_gridcell_red.xml
/malaria-app-android/res/layout/screen_gridcell_green.xml
/malaria-app-android/res/layout/fragment_welcome_dummy.xml

**Comment by harshitagupta30** [comment]
@anubhakushwaha Can I send the PR for the same? :)

**Comment by anubhakushwaha** [comment]
Yeah please check thoroughly regarding all the resources and then you can send :+1:

**Comment by harshitagupta30** [comment]
@anubhakushwaha yeah definitely! Thanks :+1:

**Comment by free4murad** [comment]
@anubhakushwaha @harshitagupta30 
I have already removed some unused resources in PR #165 
Also I have used properties in gradel which will exclude any unused resources from the final build.

**Comment by free4murad** [comment]
In PR #165 I have reduced the size of APK from 2.4MB to 1.1 MB

**Comment by anamikalovesSRK** [comment]
I can see , label free mentioned here, Can i send PR for the issue mentioned above?

**Comment by free4murad** [comment]
@anamikalovesSRK
This issue has already been solved in PR #165

**Comment by geekanamika** [comment]
@m-murad @MeepyMay @anubhakushwaha This issue can be closed. As #165 has been merged already.
